src/tsconfig-loader.ts

```diff
@@ -139,7 +139,14 @@ export function loadTsconfig(
       );
     }
 
-    return deepmerge(base, config);
+    return {
+      ...base,
+      ...config,
+      compilerOptions: {
+        ...base.compilerOptions,
+        ...config.compilerOptions
+      }
+    };
```

Comment:
Are there any other nested properties we need to override?

I wondered if there was a way to do this automatically, but I think we have to be selective about which nested properties should be merged.